manager/integration/tests/common.py

```diff
@@ -1935,9 +1937,10 @@ def wait_for_replica_scheduled(client, volume_name, to_nodes,
 
         time.sleep(RETRY_INTERVAL)
 
-    assert scheduled == expect_success
-    assert unexpect_fail == 0
-    assert len(volume.replicas) == expect_success + expect_fail
+    assert scheduled == expect_success, f" Volume = {volume}"
+    assert unexpect_fail == 0, f" Volume = {volume}"
+    assert len(volume.replicas) == expect_success + expect_fail, \
+        f" Volume = {volume}"
```

Comment:
We already has the volume name from the error trace but it is ok to print it out here too